### PR TITLE
Balsamic QC analysis-workflow flag [next BALSAMIC release]

### PR DESCRIPTION
**Pull request by rannick**
### Added
- balsamic-qc analysis workflow

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by ivadym** [comment]
@rannick
Removed BALSAMIC QC Scout upload following decisions made here:
https://docs.google.com/spreadsheets/d/1uRSX8QKmvyYWmry5qQrc3udE3zmBkFIXS1EswmsZt0g/edit#gid=0

**Comment by ivadym** [comment]
### Tests performed on a TO panel case:

#### Config case

`cg workflow balsamic-qc config-case alertcod`

Screen Shot 2022-06-28 at 15 11 12

#### Dry run

`cg workflow balsamic-qc run alertcod --dry-run`

Screen Shot 2022-06-28 at 15 12 46

#### Store

`cg workflow balsamic-qc store alertcod`

Screen Shot 2022-06-28 at 15 16 00

Screen Shot 2022-06-28 at 15 16 30


#### Deliver

`cg deliver analysis -d balsamic-qc -c alertcod`

Screen Shot 2022-06-28 at 15 16 52

File: cg/meta/workflow/balsamic_qc.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from typing import List, Optional
+
+from cg.constants import DataDelivery, Pipeline
+from cg.exc import BalsamicStartError
+from cg.meta.workflow.analysis import AnalysisAPI
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.meta.workflow.fastq import BalsamicFastqHandler
+from cg.models.cg_config import CGConfig
+from cg.store import models
+from cg.utils import Process
```
Comment (patrikgrenfeldt) [inline cg/meta/workflow/balsamic_qc.py:15]: I think you could remove all unused imports in grey: 
image

**Review by patrikgrenfeldt — APPROVED** [review]
Great, seems like you have made a very thorough job!

File: cg/meta/workflow/balsamic_qc.py (hunk 1 above)
Comment (rannick) [reply cg/meta/workflow/balsamic_qc.py]: Agreed: [e692084](https://github.com/Clinical-Genomics/cg/pull/1503/commits/e6920845da65df2243583c732b65fcf1e9f776d4)

**Pull request merged**